safeeyes/safeeyes.py

```diff
@@ -71,8 +71,8 @@ def __init__(self, system_locale, config):
             self.show_about)
         self.context['api']['enable_safeeyes'] = lambda next_break_time=-1, reset_breaks=False: \
             utility.execute_main_thread(self.enable_safeeyes, next_break_time, reset_breaks)
-        self.context['api']['disable_safeeyes'] = lambda status: utility.execute_main_thread(
-            self.disable_safeeyes, status)
+        self.context['api']['disable_safeeyes'] = lambda status=None, is_resting=False: utility.execute_main_thread(
+            self.disable_safeeyes, status, is_resting)
```

Comment:
This change was necessary due to 903d407faf58871fd84756ec76d6b5c2543f6540 not changing this.